frontend/.env.local.example

```
@@ -6,5 +6,5 @@ GOOGLE_CLIENT_ID=your_google_client_id_here
 GOOGLE_CLIENT_SECRET=your_google_client_secret_here
 
 # Cijene API Configuration
-NEXT_PUBLIC_CIJENE_API_URL=https://api.cijene.dev
-NEXT_PUBLIC_CIJENE_API_TOKEN=bearer_token
\ No newline at end of file
+CIJENE_API_URL=https://api.cijene.dev
+CIJENE_API_TOKEN=bearer_token
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The CIJENE_API_TOKEN key should go before the CIJENE_API_URL key

(UnorderedKey)